UoaWDCC/younite-web
# 74 History Component

## Context
Wanted to correctly implement history timeline functionality and make it look like the design on Figma. 

Closes
#74 

## What Changed?
- Converted vertical timeline to be horizontal
- Added alternating logic for timeline
- Created different components for images and textboxes
- Added middle vertical line for image/text components
- Ensured images are rendered properly

## How To Review
- View next/app/about/page.tsx for the main About Us page
- View components/history for all new added components
- Go to About Us page and check history timeline looks like design on Figma
